contracts/interfaces/INexusViewer.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.18;
+
+import {IAssessmentViewer} from "./IAssessmentViewer.sol";
+import {IStakingViewer} from "./IStakingViewer.sol";
+
+interface INexusViewer {
+
+  struct ClaimableNXM {
+    uint governanceRewards; // Governance rewards in NXM
+    uint assessmentRewards; // Claimable assessment reward in NXM
+    uint assessmentStake; // Claimable assessment stake in NXM
+    uint stakingPoolTotalRewards; // Total staking pool rewards in NXM
+    uint stakingPoolTotalExpiredStake; // Total staking pool expired stake in NXM
+    uint managerTotalRewards; // Pool manager total staking rewards in NXM
+    uint legacyPooledStakeRewards; // Legacy pooled staking deposits in NXM
```

Comment:
nit: `// Legacy pooled staking rewards in NXM`

Reply:
oh good catch 👍